contracts/strategies/examples/donation-voting/DonationVotingOnchain.sol

```diff
@@ -53,45 +44,24 @@ contract DonationVotingOnchain is BaseStrategy, RecipientsExtension, Native {
     /// @notice Thrown when the timestamps being set or updated don't meet the contracts requirements.
     error INVALID_TIMESTAMPS();
 
+    /// @notice Thrown when the allocation token is not allowed.
+    error TOKEN_NOT_ALLOWED();
+
     /// ================================
     /// ========== Storage =============
     /// ================================
 
-    /// @notice The start and end times for allocations
-    uint64 public allocationStartTime;
-    uint64 public allocationEndTime;
     /// @notice Cooldown time from allocationEndTime after which the pool manager is allowed to withdraw tokens.
     uint64 public withdrawalCooldown;
     /// @notice amount to be distributed. It is set during the first distribute() call and stays fixed.
     uint256 public totalPayoutAmount;
 
-    /// @notice token -> bool
-    address public allocationToken;
```

Comment:
Because of this removal I had to make some modifications on `_allocate` and `_distribute`